CMakeLists.txt

```diff
@@ -92,6 +92,7 @@ if(NOT WIN32)
 		if(NOT DEFINED PROBE_NAME)
 			set(PROBE_NAME "sysdig-probe")
 		endif()
+		string(REPLACE "-" "_" PROBE_NAME ${PROBE_NAME})
```

Comment:
Might be worth creating a `KMOD_NAME`-like variable as seen in the driver loader script instead: https://github.com/draios/sysdig/blob/6a5b84599c7ec025b639e054bd1b6092b325f148/scripts/sysdig-probe-loader#L126